src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/RetryPolicy.SqlAzureTemporaryAndIgnorableErrorDetectionStrategy.cs

```diff
@@ -20,10 +20,12 @@ public abstract partial class RetryPolicy
         /// </summary>
         public class SqlAzureTemporaryAndIgnorableErrorDetectionStrategy : ErrorDetectionStrategyBase
         {
+#pragma warning disable CA1859 // Use concrete types when possible for improved performance
```

Comment:
Would suggest just disabling this in editorconfig - and then undoing all the other type changes you did. We can address those separately if needed.